.github/workflows/liquibase-migration-check.yml

```
@@ -44,30 +55,86 @@ jobs:
+      - name: Apply database migrations
         working-directory: server/application-server
         run: mvn liquibase:update
 
-      - name: Run Liquibase diff
+      - name: Validate migration integrity
         working-directory: server/application-server
         run: mvn liquibase:diff
 
-      - name: Check for changes in the liquibase changelog
+      - name: Set up Python for ERD generation
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)